infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/StoppedState.java

```diff
@@ -25,7 +25,7 @@ public LogReplicationState processEvent(LogReplicationEvent event) {
     @Override
     public void onEntry(LogReplicationState from) {
         fsm.getAckReader().getOngoing().set(false);
-        fsm.getAckReader().markSyncStatus(LogReplicationMetadata.SyncStatus.ERROR);
+        fsm.getAckReader().markSyncStatus(SyncStatus.ERROR);
```

Comment:
what is the difference between the 'error' state and 'stopped' state for replication status?

Reply:
Replication can be stopped explicitly (nothing went wrong, it was just stopped) or could be stopped because the connection got lost but replication itself did not error out. Then error, means something went wrong with replication itself (corruption or data loss from which it can't recover).